platform/security/expansion/security-expansion-commands/pom.xml

```diff
@@ -50,11 +50,11 @@
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
+                        <Karaf-Commands>*</Karaf-Commands>
```

Comment:
Is this line needed? I see other bundles that already have the new karaf command annotations do not ahve this line.